.github/CODEOWNERS

```diff
@@ -22,6 +22,9 @@
 # rules module
 /src/packages/rules/ @MHajoha
 
+# RatSig related module
+/src/packages/ratmama @beepbeat
```

Comment:
[Scope] Modifying codeowners is out of scope for your pull request, and currently irrelevant for those without write access to the repository.  Please remove this.

Reply:
Modifying Codeownership is something that should have been in the original PR, therefore it's well within scope of this one.

 Even if it's useless since I have no write access, I still have the right to put my thumbprint on modules that have been created by me.

Reply:
I'm not going to get drawn into a debate with you about 'rights' with your code.  Once it is integrated into the project its no longer 'your code.'  That ideology is incompatible with an open-source project.

That point aside, anyone can see who contributed to the project and who authored specific commits.   What you are proposing is not the intended usage of the codeowners file.  If you had submitted a modification to the codeowners file as part of your original PR, It also would have been considered out of scope.